misc-scripts/xref_mapping/XrefParser/RefSeqCoordinateParser.pm

```diff
@@ -431,4 +328,85 @@ sub run_script {
   return 0;
 }
 
+
+
+# parses the input string $file into an hash
+# string $file is in the format as the example:
+# script:project=>ensembl,host=>ens-staging1,dbname=>homo_sapiens_core_70_37,ofhost=>ens-staging1,...
+# string until : is ignored, hash is built with keys=>values provided
+sub parse_file_string {
+  my ($self, $file_string) = @_;
+
+  $file_string =~ s/\A\w+://xms;
+
+  my @param_pairs = split( /,/x, $file_string );
+
+  my $params;
+
+  # set provided values
+  foreach my $pair ( @param_pairs ) {
+    my ($key, $value) = split( /=>/x, $pair );
```

Comment:
I would have the split pattern include optional whitespace on either side of =>, to be absolutely certain we do not end up slurping whitespace into keys or values in the event of it having suddenly begun to appear in the file string.

Reply:
this is unnecessary because if any whitespace is there FetchFiles.pm will scream before it gets here